RxKeyboard.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'RxKeyboard'
-  s.version          = '0.9.2'
+  s.version          = '0.9.3'
```

Comment:
1.0.0 - we're bumping a major version of RxSwift

Reply:
Resolved